model/database/orm.py

```diff
@@ -0,0 +1,100 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+A class responsible for creating a light weight django instance to
+access the Django ORM that defines the autoreduction database
+"""
+import os
+import sys
+import logging
+
+import django
+from django.conf import settings
+
+from utils.project.structure import get_project_root
+
+
+class DjangoORM:
+    """
+    Encapsulate Django setup and ORM access in a class
+    This should normally be used like the following:
+    orm = DjangoORM
+    orm.connect()
+    records = orm.data_model.<TableName>.objects.<Filter/OrderBy/..>
+    """
+
+    data_model = None
+    variable_model = None
+
+    @staticmethod
+    def add_webapp_path():
+        """
+        Get path to the autoreduce_webapp and add this path to the system path
+        This will enable importing of the autoreduce_webapp (and it's models)
+        """
+        path = os.path.join(get_project_root(), 'WebApp', 'autoreduce_webapp')
+        if path not in sys.path:
+            # Can't properly replicate this in a testing environment due to travis
+            # constraint on sys path access and mocking would break underlying dependency
+            # but it's simple enough we don't need to worry about testing it
+            sys.path.append(path)  # pragma: no cover
+
+    @staticmethod
+    def setup_django():
+        """
+        Use the WebApp settings to initialise a django instance that can be used for model access
```

Comment:
Maybe a note to say that "add_webapp_path should always be called before this method" or similar (if I'm remembering correctly that that is the case)